sections/Pipeline.tex

```diff
@@ -0,0 +1,34 @@
+\section{Method}\label{sec:method}
+This section describes our method on an abstract level.
+Each phase of the method is described later on in more detail. 
+
+We start with a dataset, in our case consisting of news articles from the media group Nordjyske. 
+Their primary focus is to maintain a variety of local newspapers within the North Jutland region of Denmark. 
+The data ranges from 2017, where a total of $\sim$~63.000 articles have been extracted from their database.
+
+\subsection*{Step 1: Preprocessing Phase}
+This phase applies different \gls{NLP} methods, such as stemming and removing stop words, to simplify the data set and remove redundant information.
+Details of this phase are given in \autoref{sec:prepro}.
+After finishing this phase, we are left with $\sim$~32.000 articles, which will be used in the next step.
```

Comment:
```suggestion
After finishing this phase, we are left with $\sim$~32.000 documents, which will be used in future steps.
```
We also need to figure out when we will go from calling them articles to documents.
I think it should be after preprocessing (aka. here)
Another option is to clear it up / change it in the dataset section.